context-free.md

```diff
@@ -1,22 +1,44 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+ContextFree can be used by people who want to create digital images, 
+especially those that involve repeating structures, through some simple
+programming.
```

Comment:
I agree! This is similar to what I said about contextFree, but I think you expressed it more eloquently.